tests/IResearch/ExpressionFilter-test.cpp

```diff
@@ -265,7 +265,8 @@ struct IResearchExpressionFilterSetup {
     arangodb::tests::init(true);
 
     // suppress INFO {authentication} Authentication is turned on (system only), authentication for unix sockets is turned on
-    arangodb::LogTopic::setLogLevel(arangodb::Logger::AUTHENTICATION.name(), arangodb::LogLevel::WARN);
+    // suppress WARNING {authentication} --server.jwt-secret is insecure. Use --server.jwt-secret-keyfile instead
+    arangodb::LogTopic::setLogLevel(arangodb::Logger::AUTHENTICATION.name(), arangodb::LogLevel::ERR);
```

Comment:
same

Reply:
it's safe to ignore all errors from the authentication topic anyway, since there are tests to check invalid permissions
and there are quite a few test files with this change, hence I am not sure if it's worthwhile to revert